one_fm/hooks.py

```diff
@@ -164,7 +164,7 @@
 		"on_submit": "one_fm.utils.leave_appillication_on_submit",
 		"validate": [
 			"one_fm.utils.validate_hajj_leave",
-			"one_fm.one_fm.hr_utils.validate_leave_proof_document_requirement",
+			"one_fm.one_fm.doctype.indemnity_allocation.indemnity_allocation.daily_indemnity_allocation_builder.validate_leave_proof_document_requirement",
```

Comment:
Wrong path and needs to revise this change - This is not purely related to indemnity, It is related to the Leave Type and Leave Application